beetsplug/acousticbrainz.py

```diff
@@ -41,64 +45,168 @@ def func(lib, opts, args):
         cmd.func = func
         return [cmd]
 
+    def import_task_files(self, session, task):
+        """Function is called upon beet import.
+        """
+
+        items = task.imported_items()
+        fetch_info(self._log, items)
+
 
 def fetch_info(log, items):
-    """Currently outputs MBID and corresponding request status code
+    """Currently outputs MBID and corresponding request status code.
     """
     for item in items:
         if item.mb_trackid:
             log.info('getting data for: {}', item)
 
             # Fetch the data from the AB API.
-            url = generate_url(item.mb_trackid)
-            log.debug('fetching URL: {}', url)
+            high_url = generate_url(item.mb_trackid, "/high-level")
+            low_url = generate_url(item.mb_trackid, "/low-level")
+            log.debug('fetching URL: {}', high_url)
+            log.debug('fetching URL: {}', low_url)
             try:
-                rs = requests.get(url)
+                high = requests.get(high_url)
+                low = requests.get(low_url)
             except requests.RequestException as exc:
                 log.info('request error: {}', exc)
                 continue
 
             # Check for missing tracks.
-            if rs.status_code == 404:
+            if high.status_code == 404 or low.status_code == 404:
                 log.info('recording ID {} not found', item.mb_trackid)
                 continue
 
             # Parse the JSON response.
             try:
-                data = rs.json()
+                high_data = high.json()
             except ValueError:
-                log.debug('Invalid Response: {}', rs.text)
+                log.debug('Invalid Response: {}', high.text)
```

Comment:
For these messages, we can now be more specific about what failed. "Invalid response for high-level features," for example.